index.html

```diff
@@ -0,0 +1,186 @@
+					<button type="button">HTML</button>
+					<button type="button">Boostrap</button>
+					<button type="button">Ruby</button>
+				</div>
+				<button  type="button"class="project_btn btn">See project</button>
+			</div>
+			
+			<div class="box ">
+				<h4>Profesional Art Printing Data</h4>
+				<p>A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p>
+				<div class="butons_container grid">
+					<button type="button">HTML</button>
+					<button type="button">Boostrap</button>
+					<button type="button">Ruby</button>
+				</div>
+				<button type="button" class="project_btn btn">See project</button>
+			</div>
+			
+			<div class="box ">
+				<h4>Profesional Art Printing Data</h4>
+				<p>A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p>
+				<div class="butons_container grid">
+					<button type="button">HTML</button>
+					<button type="button">Boostrap</button>
+					<button type="button">Ruby</button>
+				</div>
+				<button type="button" class="project_btn btn">See project</button>
+			</div>
+			
+			
+			<div class="box ">
+				<h4>Profesional Art Printing Data</h4>
+				<p>A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p>
+				<div class="butons_container grid">
+					<button type="button">HTML</button>
+					<button type="button">Boostrap</button>
+					<button type="button">Ruby</button>
+				</div>
+				<button type="button"  class="project_btn btn">See project</button>
+			</div>
+			
+			<div class="box ">
+				<h4>Profesional Art Printing Data</h4>
+				<p>A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p>
+				<div class="butons_container grid">
+					<button type="button">HTML</button>
+					<button type="button">Boostrap</button>
+					<button type="button">Ruby</button>
+				</div>
+				<button  type="button" class="project_btn btn">See project</button>
+			</div>
+			
+			<div class="box ">
+				<h4>Profesional Art Printing Data</h4>
+				<p>A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p>
+				<div class="butons_container grid">
+					<button type="button" >HTML</button>
+					<button type="button" >Boostrap</button>
+					<button type="button" >Ruby</button>
+				</div>
+				<button type="button" class="project_btn btn">See project</button>
+			</div>
+		</div>
+	</section>
+	<section class="about_container">
+		<section class="about">
+			<h1>About me</h1>
+		<p>Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
+		<button type="button" >Get my resume</button>
+		</section>
```

Comment:
Your design
![image](https://user-images.githubusercontent.com/78251927/229278531-8c9d1f10-1ad9-4330-8109-96ccb1d90cf3.png)

Figma
![image](https://user-images.githubusercontent.com/78251927/229278574-54d34489-29e6-4d80-9e73-5f3d033ffca2.png)

- [ ] Kindly adjust the section margins (top, bottom, left, right) defined in the design so that our design matches the template as much as possible.
- [ ] Kindly change the color of `Get my resume` inside the `Button` to `white` to match the design.